include/crow/http_response.h

```diff
@@ -124,6 +124,11 @@ namespace crow
             }
         }
 
+        void end(const int& _code) {
+            code = _code;
+            end();
+        }
+
```

Comment:
This is unrelated to the PR, and as far as I know not exactly a problem. It's probably best not to clutter the response class as much.

Reply:
No problem @The-EDev  i will remove this. I am still not familiar with the complete codebase that's why i added this too from the old repository PR.

Reply:
Also it might be best to change the examples to reflect this change?

Reply:
Also, trying to build the examples without a request parameter resulted in `error: call of ‘(crow::TaggedRule<long int, long int>) (main()::<lambda(crow::response&, int, int)>)’ is ambiguous`

Reply:
okay i will see this and will also add an example to reflect these changes

Reply:
You probably don't need to add an example, rather change the existing ones that have a request that's not being used

Reply:
yes, will modify the existing one